packages/esm-service-queues-app/src/metrics/metrics-cards/patients-seen.extension.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import MetricsCard from './metrics-card.component';
+import { useConfig } from '@openmrs/esm-framework';
+import { useActiveVisits } from '../metrics.resource';
+import { useSelectedService } from '../../helpers/helpers';
+import { useQueueEntries } from '../../hooks/useQueueEntries';
+import { type ConfigObject } from '../../config-schema';
+import { useUniquePatientsWithAppointmentsCount } from '../../hooks/useUniquePatientsWithAppointmentsCount';
+
+export default function PatientsSeenExtension() {
+  const { t } = useTranslation();
+  const { showPatientSeenMetricsCard } = useConfig<ConfigObject>();
+  const currentService = useSelectedService();
+  const { activeVisitsCount } = useActiveVisits();
+  const { count: todaysAppointmentsCount } = useUniquePatientsWithAppointmentsCount();
+
+  const unscheduledPatientsCount = Math.max(activeVisitsCount - todaysAppointmentsCount, 0);
```

Comment:
Should we be concerned about the mixed scoping in this computation?
   
  - `activeVisitsCount` is location-scoped (all services at current location)
  - `todaysAppointmentsCount` is global (all appointments across locations/services)

This might produce inaccurate metrics when multiple services operate at the same location. Should "unscheduled" should mean "service-specific unscheduled" vs "location-wide unscheduled"?

Reply:
sure ,, thanks @denniskigen i have scoped `todaysAppointmentsCount` to location specific appointments because unscheduled should be those who visited a particular location clinic , checked in but with no appointments